CONTRIBUTING.md

```diff
@@ -87,6 +87,18 @@ This repository follows the [PSR-2 standard](http://www.php-fig.org/psr/psr-2/)
 you must follow these rules.
 
 
+## Coding style
```

Comment:
i would make this a sub-title of "Coding Standards" and call it "Additional Coding Style"